_data/sites.json

```
@@ -10252,6 +10262,7 @@
         "url": "https://account.protonvpn.com/account",
         "difficulty": "easy",
         "notes": "Click on the Delete Account button and follow the steps.",
+        "notes": "Clique em Delete Account button e siga as instruções.",
```

```suggestion
        "notes": "Clique em Delete Account button e siga as instruções.",
```
This should be "notes_pt_br”

---

Ops, my missing. Now that's corrected. Thanks.